src/coreclr/runtime.proj

```diff
@@ -103,7 +102,7 @@
       <_CoreClrBuildArg Include="-cmakeargs &quot;-DCLR_CMAKE_ESRP_CLIENT=$(DotNetEsrpToolPath)&quot;" />
     </ItemGroup>
 
-    <ItemGroup Condition="'$(TargetsBrowser)' == 'true'">
+    <ItemGroup Condition="'$(KeepNativeSymbols)' == 'true' or '$(TargetsWindows)' != 'true'">
```

Comment:
This condition replaces `$(KeepNativeSymbols)' != 'false'. I'm still not a fan as I think we want it to be opt in except for (I think) wasm that wants it on explicitly. 

Are there other platforms where want this on by default?

Reply:
I don't think we have other platforms. Symbols in separate files remove > 50% from the binary size, e.g.

```sh
$ dotnet10 publish -o dist -c Release
$ du -sh dist/*
1.3M	dist/aot7
2.3M	dist/aot7.dbg

$ dotnet10 publish -o dist -c Release -p:StripSymbols=false
$ du -sh dist/*
3.5M	dist/aot7
```
Production machine will only need a binary. Symbols are deemed optional; downloaded only when investigating issues.